packages/rancher-istio/charts/app-readme.md

```diff
@@ -4,20 +4,65 @@ Our [Istio](https://istio.io/) installer wraps the istioctl binary commands in a
 * **[Kiali](https://kiali.io/)**: Used for graphing traffic flow throughout the mesh
 * **[Jaeger](https://www.jaegertracing.io/)**: A quick start, all-in-one installation used for tracing distributed systemm. This is not production qualified, please refer to jaeger documentation to determine which installation you may need instead.
 
-### Known Issues
+## Known Issues
 
-**Airgapped Environments**
+#### Airgapped Environments
 If you are using this chart in an airgapped environment, you will not be able to upgrade. This is because the `istioctl` upgrade command reaches out to an external repo and it is not configurable. We are tracking the fix for this issue [here](https://github.com/rancher/rancher/issues/33402)
```

Comment:
We'll have to remove this when other PR goes in, or adjust it to say the final fix is not in, that issue actually links to long term fix. I'd say we just remove and tell them to use workaround in values.yaml.